puppet/modules/jenkins_job_builder/files/centos.org/pipelines/test/foreman/nightly-centos7.groovy

```diff
@@ -1,12 +1,4 @@
 def playBookVars() {
-    playBook = [
-      'boxes': ['pipeline-foreman-server-nightly-centos7', 'pipeline-foreman-smoker-nightly-centos7'],
-      'pipeline': 'install_pipeline.yml',
-      'extraVars': [
-        'pipeline_version': 'nightly',
-        'pipeline_os': 'centos7',
-        'pipeline_type': 'foreman'
-      ]
-    ]
+    playBook = pipelineVars(action: 'install', type: 'foreman', version: 'nightly', os: 'centos7')
```

Comment:
If I just had return here with the additional variable, it all failed. Not sure what that's about.